<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu_window</class>
 <widget class="QMainWindow" name="menu_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QFrame" name="menu_frame">
      <property name="styleSheet">
       <string notr="true">QFrame {
	border-radius: 7px;
	color: rgb(62, 231, 41);
	background-color: rgb(40, 109, 50);
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="name_label">
       <property name="geometry">
        <rect>
         <x>-190</x>
         <y>20</y>
         <width>661</width>
         <height>61</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>50</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Scrabble&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="exit_button">
       <property name="geometry">
        <rect>
         <x>489</x>
         <y>5</y>
         <width>14</width>
         <height>14</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(255, 40, 54);
	border-radius: 7px;
	border: none
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="emoji_label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>440</y>
         <width>151</width>
         <height>121</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/images/3x.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QFrame" name="game_mode_frame">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>110</y>
         <width>231</width>
         <height>311</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame {
background-color: rgb(22, 60, 27);
}
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="game_mode_label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: rgb(226, 224, 158);

}</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Tryb gry&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="hotseat_mode_label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>240</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	
	color: rgb(204, 204, 143);
}</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; font-style:italic;&quot;&gt;hot seat&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="ai_mode_label">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>240</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	
	color: rgb(204, 204, 143);
}</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; font-style:italic;&quot;&gt;kontra SI&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="game_mode_desc">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>70</y>
          <width>161</width>
          <height>81</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget{
	
	color: rgb(204, 204, 143);
}</string>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:14pt; font-weight:700; font-style:italic;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:13pt;&quot;&gt;wybierz jeden z trybów gry i zmierz się ze znajomymi lub automatem!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="hotseat_button">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>171</y>
          <width>101</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: transparent;
	image: url(:/newPrefix/images/gamer.png);
	
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="vs_si_button">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>170</y>
          <width>111</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: transparent;
	image: url(:/newPrefix/images/swords.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="ranking_frame">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>110</y>
         <width>171</width>
         <height>221</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame {
background-color: rgb(22, 60, 27);
}
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="ranking_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>15</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	
	color: rgb(226, 224, 158);
}</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Top 5&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>154</width>
          <height>161</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="rank_vert_layout">
         <item>
          <layout class="QHBoxLayout" name="player1_h_lay">
           <property name="spacing">
            <number>-1</number>
           </property>
           <item>
            <widget class="QLabel" name="p1_name_label">
             <property name="font">
              <font/>
             </property>
             <property name="text">
              <string>pholder</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="p1_score_label">
             <property name="font">
              <font/>
             </property>
             <property name="text">
              <string>pholder</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="player4_h_lay">
           <item>
            <widget class="QLabel" name="p2_name_label">
             <property name="font">
              <font/>
             </property>
             <property name="text">
              <string>pholder</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="p2_score_label">
             <property name="font">
              <font/>
             </property>
             <property name="text">
              <string>pholder</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="player5_h_lay">
           <item>
            <widget class="QLabel" name="p3_name_label">
             <property name="font">
              <font/>
             </property>
             <property name="text">
              <string>pholder</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="p3_score_label">
             <property name="font">
              <font/>
             </property>
             <property name="text">
              <string>pholder</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="player3_h_lay">
           <item>
            <widget class="QLabel" name="p4_name_label">
             <property name="font">
              <font/>
             </property>
             <property name="text">
              <string>pholder</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="p4_score_label">
             <property name="font">
              <font/>
             </property>
             <property name="text">
              <string>pholder</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="player2_h_lay">
           <item>
            <widget class="QLabel" name="p5_name_label">
             <property name="font">
              <font/>
             </property>
             <property name="text">
              <string>pholder</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="p5_score_label">
             <property name="font">
              <font/>
             </property>
             <property name="text">
              <string>pholder</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QFrame" name="misc_frame">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>350</y>
         <width>171</width>
         <height>201</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame {
background-color: rgb(22, 60, 27);
}
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="misc_label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	
	color: rgb(226, 224, 158);
}</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Opcje&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="logout_button">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>150</y>
          <width>71</width>
          <height>32</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: transparent;
image: url(:/newPrefix/images/returnleft.png);
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="help_button">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>70</y>
          <width>71</width>
          <height>32</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: transparent;
image: url(:/newPrefix/images/lookingglass.png);
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="account_button">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>110</y>
          <width>71</width>
          <height>32</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color:transparent;
	image: url(:/newPrefix/images/edit.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="help_label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>75</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	color: rgb(226, 225, 158);
}</string>
        </property>
        <property name="text">
         <string>pomoc</string>
        </property>
       </widget>
       <widget class="QLabel" name="acc_manage_label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>115</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	color: rgb(226, 225, 158);
}</string>
        </property>
        <property name="text">
         <string>konto</string>
        </property>
       </widget>
       <widget class="QLabel" name="logout_label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>155</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	color: rgb(226, 225, 158);
}</string>
        </property>
        <property name="text">
         <string>wyloguj</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../src_qt.qrc"/>
 </resources>
 <connections/>
</ui>
